src/Symfony/Component/Process/ProcessBuilder.php

```diff
@@ -221,8 +221,10 @@ public function getProcess()
 
         $options = $this->options;
 
-        $arguments = $this->prefix ? array_merge(array($this->prefix), $this->arguments) : $this->arguments;
-        $script = implode(' ', array_map(array(__NAMESPACE__.'\\ProcessUtils', 'escapeArgument'), $arguments));
+        $prefix = $this->prefix ? $this->prefix : '';
+        $arguments = implode(' ', array_map(array(__NAMESPACE__.'\\ProcessUtils', 'escapeArgument'), $this->arguments));
+
+        $script = $prefix && $arguments ? $prefix.' '.$arguments : ($prefix ? $prefix : $arguments);
```

Comment:
I don't think so as you could have just a prefix and no arguments, so a trailing space would be present